crates/testing-framework/src/transform/walker.rs

```diff
@@ -57,7 +58,9 @@ impl Transformer {
             self.results.insert(call_id, service.clone());
 
             match &mut call.triplet.1 {
-                Sexp::String(ref mut value) => value.push_str(&format!("..{}", call_id)),
+                Sexp::String(ref mut value) => {
+                    let _ = write!(value, "..{}", call_id);
```

Comment:
Shoudn't it be `.unwrap()`?

Reply:
Hmm, looks like it can't throw an error, but ok, let's add unwrap here